packages/vinxi-mdx/package.json

```diff
@@ -31,13 +38,14 @@
     "vfile": "^5.0.0"
   },
   "devDependencies": {
-    "@mdx-js/mdx": "2.3.0",
+    "@mdx-js/mdx": "3.1.0",
```

Comment:
v3 has been around since the end of 2023, so I think it is safe to use the latest.